Provide example of how to read specific message types from enum

Personally, as a newer rust user, I'm struggling with the lack of examples in this repository, though it works great for parsing.

Specifically, when trying to match an entry's message to it's type, I get:

```
error[E0532]: expected unit struct/variant or constant, found struct variant `MrtMessage::RIB_IPV4_UNICAST`
  --> src/main.rs:26:33
   |
26 |                                 MrtMessage::RIB_IPV4_UNICAST => println!("{:?}", entry.message),
   |                                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^ did you mean `MrtMessage::RIB_IPV4_UNICAST { /* fields */ }`?
help: possible better candidate is found in another module, you can import it into scope
```

This is part of:

```
use std::path::Path;
use std::fs::File;
use std::io::BufReader;

use libflate::gzip::Decoder;
use mrt::{MrtFile, MrtSubType, MrtMessage};
use indicatif::ProgressBar;


fn main() {
    {
        let bar = ProgressBar::new(245);
        let p = Path::new("rib_dump");
        for entry in p.read_dir().expect("read_dir call failed") {
            bar.inc(1);
            if let Ok(entry) = entry {
                let f = File::open(entry.path()).unwrap();
                let decoder = Decoder::new(BufReader::new(f)).unwrap();
                let mrtfile = MrtFile::new(decoder);

                println!("Beginning to read");
                for entry in mrtfile {
                    match entry.mrt_header.mrt_subtype {
                        MrtSubType::RIB_IPV4_UNICAST => {
                            match entry.message {
                                MrtMessage::RIB_IPV4_UNICAST => println!("{:?}", entry.message),
                                _ => continue,
                            }
                        },
                        _ => continue,
                    }
                }
            }
        }
        bar.finish();
    }
    println!("End scope!")
}
```

Are you able to provide an example of how this should work?

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -550,11 +550,20 @@ pub fn read_file_complete(file: File) -> Result<Vec<MrtEntry>, &'static str> {
 /// # Example
 /// ```
 /// use std::fs::File;
+/// use mrt::MrtMessage;
 /// let f = File::open("example_data/openbgpd_rib_table-v2").unwrap();
 /// let mrtfile = mrt::MrtFile::new(f);
 ///
-/// for entry in mrtfile {
-///     println!("{:?}", entry);
+/// for mrt_entry in mrtfile {
+///   match mrt_entry.message {
+///     MrtMessage::RIB_IPV4_UNICAST {header, entries} => {
+///       println!("header: {:?}", header);
+///       for entry in entries {
+///         println!("entry: {:?}", entry);
+///       }
+///     },
+///     _ => continue,
+///   }
 /// }
 /// ```
 //#[derive(Debug)]

